src/HttpServer.php

```diff
@@ -117,14 +117,22 @@ public function isChunkedEncodingActive(Request $request)
     public function handleRequest(ConnectionInterface $connection, Request $request)
     {
         $callback = $this->callback;
-        $response = $callback($request);
+
+        try {
+            $response = $callback($request);
+        } catch (\Exception $exception) {
+            $connection->write(RingCentral\Psr7\str(new Response(500)));
```

Comment:
I suppose an empty body is intended here? Note how many user agents (browsers) usually don't show HTTP headers etc.